airship_tempest_plugin/services/shipyard/json/document_staging_client.py

```diff
@@ -52,13 +56,30 @@ def get_configdocs(self, collection_id=None):
         resp, body = self.get('configdocs/%s' % collection_id)
         self.expected_success(200, resp.status)
         body = json.loads(body)
-        return rest_client.ResponseBody(resp, body)
+        if isinstance(body, list):
+            return rest_client.ResponseBodyList(resp, body)
+        else:
+            return rest_client.ResponseBody(resp, body)
+
+    def get_configdocs_version(self, collection_id=None, version_arg=None):
+        resp, body = self.get('configdocs/%s?version=%s' %
+                              (collection_id, version_arg))
+        self.expected_success(200, resp.status)
+        return rest_client.ResponseBodyData(resp, body)
 
     def get_renderedconfigdocs(self):
         resp, body = self.get('renderedconfigdocs')
         self.expected_success(200, resp.status)
         body = json.loads(body)
-        return rest_client.ResponseBody(resp, body)
+        if isinstance(body, list):
```

Comment:
is this needing when doing a get on a specific artifact/resource?

Reply:
This is returning a List only; therefore used "ResponseBodyList".